automol/util/vector.py

```diff
@@ -270,160 +257,155 @@ def _local_axes(xyz1=(0.0, 0.0, 0.0), xyz2=(0.0, 0.0, 1.0), xyz3=(0.0, 1.0, 0.0)
     return (x_ax, y_ax, z_ax)
 
 
-def distance(xyz1, xyz2):
+def distance(xyz1: Sequence | np.ndarray, xyz2: Sequence | np.ndarray) -> float:
     """Measure the distance between points.
 
     :param xyz1: 3D vector to point 1
-    :type xyz1: tuple, list, or numpy nd.array
     :param xyz2: 3D vector to point 2
-    :type xyz2: tuple, list, or numpy nd.array
-    :rtype: float
+    :return: Distance
     """
-
-    dist = numpy.linalg.norm(numpy.subtract(xyz1, xyz2))
+    dist = np.linalg.norm(np.subtract(xyz1, xyz2))
 
     return dist
 
 
-def angle(xyz1, xyz2, orig_xyz=(0.0, 0.0, 0.0)):
+def angle(
+    xyz1: Sequence | np.ndarray,
+    xyz2: Sequence | np.ndarray,
+    orig_xyz: Sequence | np.ndarray = (0.0, 0.0, 0.0),
+) -> float:
     """Measure the angle inscribed by three atoms.
 
     :param xyz1: 3D vector to point 1
-    :type xyz1: tuple, list, or numpy nd.array
     :param xyz2: 3D vector to point 2
-    :type xyz2: tuple, list, or numpy nd.array
     :param xyz3: 3D vector to point 3
-    :type xyz3: tuple, list, or numpy nd.array
-    :rtype: float
+    :Note: there is no xyz3
+    :param orig_xyz: Origin of coordinate system `xyz1` and `xyz2` are in
```

Comment:
Instead, let's say "Origin to measure the vector angle from", since they are technically in the standard coordinate system.